### ISSUE TITLE
Modify highlight colour to have a higher contrast

### ISSUE THREAD

**Issue by ghost** [dracula/rstudio#15]
When searching for a word or even highlight, the default colour doesn't provide enough contrast. It's light gray.
Would it be possible to improve? Thank you

<img width="352" alt="Screenshot 2023-03-02 at 8 51 50 PM" src="https://user-images.githubusercontent.com/122058316/222611387-5e7a456e-0aab-46a4-a849-040553dff4f8.png">

**Comment by ghost** [comment]
The screenshot wasn't too representative due to the comments. Here is another one.
<img width="352" alt="Screenshot 2023-03-02 at 8 53 33 PM" src="https://user-images.githubusercontent.com/122058316/222611657-680174e3-1210-40ae-9085-37aff8094123.png">

**Comment by kputschko** [comment]
Agreed. I can never tell what I'm highlighting when using Dracula theme!
